sources/libClangSharp/ClangSharp.cpp

```diff
@@ -4693,23 +4693,51 @@ CXCursor clangsharp_Cursor_getVBase(CXCursor C, unsigned i) {
     return clang_getNullCursor();
 }
 
-int64_t clangsharp_Cursor_getVtblIdx(CXCursor C) {
+namespace {
```

Comment:
nit: I'd likely put this at the top with the other helper functions (before `clangsharp_Cursor_getArgument`) and not put it into a namespace.

Reply:
Are you OK with exporting all the symbols on non-MSVC platforms because they are not marked as hidden? All the helper functions (e.g. `isAttr()`) are currently exported. I can fix that, if you like.

Reply:
There isn't any real harm to them being exported. If a power user wants to take a dependency on them, they can.

I'd also be fine with them being explicit and matching what MSVC exports.

Reply:
There are no declarations for these functions in the header file, those who want to use them have to modify sources first. I'd say it is perfectly OK to hide them from the export table by default.